python/RDS_ENHANCED_MONITORING_ENABLED/RDS_ENHANCED_MONITORING_ENABLED.py

```diff
@@ -0,0 +1,395 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+
+'''
+
+#####################################
+##           Gherkin               ##
+#####################################
+
+Description:
+  Checks whether enhanced monitoring is enabled for Amazon RDS instances.
+Trigger:
+  Configuration Changes
+Reports on:
+  AWS::RDS::DBInstance
+Rule Parameters:
+ monitoringInterval
+ (Optional) An integer value in seconds between points when Enhanced Monitoring metrics are collected for the DB instance. Valid Values are 1, 5, 10, 15, 30, 60.
+
+Scenarios:
+  Scenario: 1
+     Given: Value for the rule parameter 'monitoringInterval' is invalid.
+      Then: Return ERROR
+  Scenario: 2
+     Given: 'monitoringInterval' is '0' in configuration item of the Amazon RDS instance.
+      Then: Return NON_COMPLIANT
+  Scenario: 3
+     Given: Value for the rule parameter 'monitoringInterval' is provided and is valid
+       And: 'monitoringInterval' in configuration item of the Amazon RDS instance does not match the rule parameter value
+      Then: Return NON_COMPLIANT
+  Scenario: 4
+     Given: Value for the rule parameter 'monitoringInterval' is not provided
+       And: 'monitoringInterval' is a non-zero value in configuration item of the Amazon RDS instance.
+      Then: Return COMPLIANT
+  Scenario: 5
+     Given: Value for the rule parameter 'monitoringInterval' is provided and is valid
+       And: 'monitoringInterval' in configuration item of the Amazon RDS instance matches the rule parameter value
+      Then: Return COMPLIANT
+
+'''
+
+import json
+import sys
+import datetime
+import boto3
+import botocore
+
+try:
+    import liblogging
+except ImportError:
+    pass
+
+##############
+# Parameters #
+##############
+
+# Define the default resource to report to Config Rules
+DEFAULT_RESOURCE_TYPE = 'AWS::RDS::DBInstance'
+
+# Set to True to get the lambda to assume the Role attached on the Config Service (useful for cross-account).
+ASSUME_ROLE_MODE = False
+
+# Other parameters (no change needed)
+CONFIG_ROLE_TIMEOUT_SECONDS = 900
+
+#############
+# Main Code #
+#############
+
+def evaluate_compliance(event, configuration_item, valid_rule_parameters):
+    if int(configuration_item['configuration']['monitoringInterval']) == 0:

```

Comment:
Comment before each if block to improve readability

Reply:
Done